pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -14,6 +14,7 @@
 
 from pandas._libs.tslibs import conversion
 from pandas._libs.tslibs.timezones import dateutil_gettz as gettz, get_timezone
+import pandas.compat as compat
```

Comment:
when only using one thing from e.g. compat, i usually prefer to do `from pandas.compat import X`.  Not a big deal

Reply:
I'm on the fence on that because I just thought I could save an extra diff in the future.
I don't mind replacing it with ```from pandas.compat import PY38```